core/scripts/server.js

```diff
@@ -93,7 +94,8 @@ app.get('*', (req, res) => {
     .pipe(res)
 })
 
-const port = process.env.PORT || 3000
+const port = process.env.PORT || config.port
+const host = config.host || 'localhost'
```

Comment:
Please keep it consistent and allow the user to set host via process.env.HOST as well